Other/Various/Adam1a1Quests.cs

```diff
@@ -0,0 +1,61 @@
+/*
+name: Poison Forest Story
+description: This will finish the Poison Forest Story.
+tags: story, quest, poison-forest
+*/
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/CoreFarms.cs
+
+using Skua.Core.Interfaces;
+
+public class Adam1a1Quest
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+    public CoreAdvanced Adv = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        doall();
+
+        Core.SetOptions(false);
+    }
+
+    public void doall()
+    {
+        if (Bot.Quests.IsUnlocked(8014))
```

Comment:
please use Core.isCompletedBefore(idgoeshere); instead

Reply:
and this is not how story scripts are made.

we use KillQuest/MapItemQuest/BuyItemQuest which the fucution itself already has a check and you can just do all say..
line 34-39 in less lines and with more checks.

Examples:
Core.KillQuest(questid, "map", "Mob");
Core.MapItemQuest(questid, "Map", "mapitemid", quant(1 is default so leave quant blank if its 1)
CoreStory.BuyQuest(QuestID, MapName, ShopID, ItemName, quant)

Reply:
please do this for the rest here